getting_started/setup_vm/D-oe.yml

```diff
@@ -1,31 +1,20 @@
 - hosts: all
   vars:
     workspace: "/tmp/"
-    oe_ver: "0.5.2"
+    oe_ver: "0.6.0"
     oe_src: "oe-{{ oe_ver }}.tar.gz"
     oe_prefix: "/opt/openenclave"
     everest_ver: "mbedtls-snapshot-7"
-    everest_src: "{{ everest_ver }}.tar.gz"
-    headers:
-      - endian.h
-      - stdbool.h
-      - inttypes.h
 
   tasks:
 
   - name: Download OpenEnclave source
     get_url:
-      url: https://github.com/openenclave/openenclave/tarball/489404f2bdfdd3e07e0f150937151ae78392e8ed
+      url: https://github.com/achamayou/openenclave/tarball/5096481ca0859940eba75e166b3764178f67482f
```

Comment:
Might be worth a `TODO` 😄

Reply:
This needs a PR against OE to be resolved, I'll add a TODO for now

Reply:
Filed https://github.com/openenclave/openenclave/pull/2100